lib/apivore/rspec_builder.rb

```diff
@@ -32,7 +32,7 @@ def apivore_build_path(path, data)
           raise URI::InvalidURIError, "No substitution data found for {#{key}} to test the path #{path}.\nAdd it via an:\n  apivore_setup '<path>', '<method>', '<response>' do\n    { '#{key}' => <value> }\n  end\nblock in your specs.", caller
         end
       end
-      path
+      path + (data['_query_string'] ? "?#{data['_query_string']}" : '')
```

Comment:
I see why this is causing problems. ```data``` might not exist here. L29 uses ```if data && data[key]``` for the conditional.